org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/graphql/Parser.java

```diff
@@ -90,7 +89,7 @@ public static Package parseJson(InputStream source) throws EGraphQLException, IO
     int col;
   }
   
-  private Reader reader;
+  protected Reader reader;
```

Comment:
why these changes? If all you're doing is making something protected instead of private.. what is happening elsewhere?